sources/packages/api/src/route-controllers/application/application.controller.ts

```diff
@@ -260,6 +260,17 @@ export class ApplicationController extends BaseController {
         `Assessment for the application id ${applicationId} was not calculated.`,
       );
     }
+    //Disbursement data is populated with dynamic key in a defined pattern to be compatible with form table.
+    const disbursementDetails = {};
+    application.disbursementSchedules.forEach((schedule, index) => {
+      disbursementDetails["disbursement" + (index + 1) + "Date"] =
```

Comment:
I would say that we have been using more string interpolation instead of concatenation with "+".

Reply:
Please consider creating a variable to keep the part being used as a disbursement identifier to keep it as one single point for a change.
```
const disbursementId = `disbursement${index + 1}`
```

Reply:
Code change has been made.

Reply:
The first change related to the concatenation was made. The second one was not changed. The changed would be as shown below.
```javascript
const disbursementDetails = {};
application.disbursementSchedules.forEach((schedule, index) => {
  const disbursementKey = `disbursement${index + 1}`;
  const disbursementDateKey = `${disbursementKey}Date`;
  disbursementDetails[disbursementDateKey] = dateString(
    schedule.disbursementDate,
  );
  schedule.disbursementValues.forEach((disbursement) => {
    const disbursementValueKey = `${disbursementKey}${disbursement.valueCode.toLowerCase()}`;
    disbursementDetails[disbursementValueKey] = disbursement.valueAmount;
  });
});
```